v1/client/api-exported.yaml

```diff
@@ -329,6 +329,56 @@ paths:
           $ref: '#/components/responses/400Response'
         '500':
           $ref: '#/components/responses/500Response'
+  '/api/v1/lifeline/{object_reference}/original-requests':
+    get:
+      summary: Api-request by object
+      description: >
+        Gets an array of api-request that have ever changed the state of the
+        object. `object_reference` as a path parameter.
+
+
+        Optionally, specify filtering, sorting, and pagination parameters. For
+        more information, refer to the [filtering, pagination,
+        sorting](#section/Insolar-Explorer-API-documentation/Filtering-pagination-sorting)
+        section.
+      operationId: api_request_by_object
+      tags:
+        - Request
+      parameters:
+        - $ref: '#/components/parameters/object_reference_path'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offsetParam'
+        - $ref: '#/components/parameters/from_index'
+        - $ref: '#/components/parameters/sort_by_index'
+        - $ref: '#/components/parameters/pulse_number_gt'
+        - $ref: '#/components/parameters/pulse_number_lt'
+        - $ref: '#/components/parameters/timestamp_gte'
+        - $ref: '#/components/parameters/timestamp_lte'
+      responses:
+        '200':
+          $ref: '#/components/responses/APIRequestResponse'
+        '400':
+          $ref: '#/components/responses/400Response'
+        '500':
+          $ref: '#/components/responses/500Response'
+  '/api/v1/requests/{request_reference}/result':
+    get:
+      summary: Result
+      description: Gets result by `request_reference` or api-request.
+      operationId: result
+      tags:
+        - Request
+      parameters:
+        - $ref: '#/components/parameters/request_reference_path'
+      responses:
+        '200':
+          $ref: '#/components/responses/resultResponse'
+        '400':
+          $ref: '#/components/responses/400Response'
+        '404':
+          $ref: '#/components/responses/404Response'
+        '500':
+          $ref: '#/components/responses/500Response'
```

Comment:
line 391 `* Record—record reference` - do we need to change description to expand the meaning of record type? (request, original request, result, state)

Reply:
Request, original request, state -- add in description.
Result - not need search